services/web/client/source/class/qxapp/About.js

```diff
@@ -19,33 +19,44 @@ qx.Class.define("qxapp.About", {
   members: {
     __populateEntries: function() {
       // All these items and versions should be red from a file
-      this.add(this.__createEntry("oSPARC UI", "3.38"));
+      this.add(this.__createEntry("oSPARC UI", "3.38", "https://github.com/ITISFoundation/osparc-simcore"));
       this.add(new qx.ui.core.Spacer(null, 10));
+      this.add(this.__createEntry("qooxdoo-compiler", qx.core.Environment.get("qx.compilerVersion"), "https://github.com/qooxdoo/qooxdoo-compiler"));
       let libInfo = qx.core.Environment.get("qx.libraryInfoMap");
       if (libInfo) {
         for (let key in libInfo) {
           let lib = libInfo[key];
           this.add(this.__createEntry(lib.name || "unknown library", lib.version || "unknown-version"));
```

Comment:
you can add `lib.homepage` to have the link

Reply:
Schön!